v2/googlecloud-to-googlecloud/pom.xml

```diff
@@ -19,6 +19,23 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <repositories>
+    <repository>
+      <id>artifact-registry</id>
+      <url>${env.BEAM_ARTIFACT_REGISTRY_URI}</url>
```

Comment:
I wonder if we should make it clear that we're not planning to submit pom.xml changes once CDC is merged into main.

Reply:
I agree, however this is more of a hack used for implementing / testing this new template. It won't actually make it to the main fork / main branch.